ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -105,6 +110,10 @@
   public void start() {
     try {
       udfLoader.load();
+      ProcessingLogServerUtils.maybeCreateProcessingLogTopic(
+          serviceContext.getTopicClient(),
+          processingLogConfig,
+          ksqlConfig);
```

Comment:
Is there anything that prevents a user from configuring `processing.log.stream.auto.create=on` in headless mode? Not sure why someone would want to do this (unless they wanted to write queries against the processing log stream...?) but in the event that someone tries, silent failure seems like poor UX. 

Then again, the docs make this pretty clear: "If you are bringing up a new interactive mode KSQL cluster, you can configure KSQL to set up a log stream automatically..." Maybe italicize "interactive" to be even more explicit?

Reply:
> Not sure why someone would want to do this (unless they wanted to write queries against the processing log stream...?)

Yeah I think that's the main reason - e.g. you could write a query to count deserialization problems to take some action downstream if they cross a threshold. Or longer term we could expose the log to udfs so users could do something similar for error events that are meaningful to them.

> silent failure seems like poor UX.

Fair point. I'll add a log message and update the docs tp make this more clear.